pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: catcher
-description: Plugin for error catching. Allows handling errors when they're not catched by developer. Plugin provides multiple handlers for errors.
+description: Plugin for error catching. Allows handling errors when they're not caught by developer. Plugin provides multiple handlers for errors.
```

Comment:
Could maybe even get fixed to
`Plugin for error catching. Allows handling errors when they're not caught by the developer and provides multiple handlers for errors.`
Then, some other grammar mistakes are fixed, too.

Reply:
Fair enough. If I'm doing more than just changing the one word then a full re-write of the description is probably not a bad idea either. I just saw it in the pub.dev description and "catched" caught my eye ;D.